<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Entity Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="EntitiesTab">
           <attribute name="title">
            <string>Entities</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTableView" name="entityTableView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CostCentersTab">
           <attribute name="title">
            <string>Cost Centers</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableView" name="costCenterTableView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="AccountsTab">
           <attribute name="title">
            <string>Accounts</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTableView" name="accountsTableView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="TrialBalanceTab">
           <attribute name="title">
            <string>Trial Balance</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTableView" name="trialBalanceTableView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="AdjustmentsTab">
           <attribute name="title">
            <string>Adjustments</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QTableView" name="adjustmentsTableView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="errorTextEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose_2"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionAdd_entity"/>
    <addaction name="actionDelete_selected_entity"/>
    <addaction name="actionAdd_cost_center"/>
    <addaction name="actionDelete_selected_cost_center"/>
    <addaction name="actionAdjustments"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionEntities"/>
    <addaction name="actionCost_Centers"/>
    <addaction name="actionAccounts"/>
    <addaction name="actionTrial_Balance"/>
    <addaction name="actionAdjustments_2"/>
   </widget>
   <widget class="QMenu" name="menuConsolidation">
    <property name="title">
     <string>Consolidation</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionAudit_adjustments"/>
    <addaction name="actionAudit_Trial_Balance"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExport"/>
   <addaction name="menuConsolidation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAdd_entity">
   <property name="text">
    <string>Entities</string>
   </property>
  </action>
  <action name="actionDelete_selected_entity">
   <property name="text">
    <string>Cost Centers</string>
   </property>
  </action>
  <action name="actionAdd_cost_center">
   <property name="text">
    <string>Accounts</string>
   </property>
  </action>
  <action name="actionDelete_selected_cost_center">
   <property name="text">
    <string>Trial Balance</string>
   </property>
  </action>
  <action name="actionAdjustments">
   <property name="text">
    <string>Adjustments</string>
   </property>
  </action>
  <action name="actionEntities">
   <property name="text">
    <string>Entities</string>
   </property>
  </action>
  <action name="actionCost_Centers">
   <property name="text">
    <string>Cost Centers</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
  </action>
  <action name="actionTrial_Balance">
   <property name="text">
    <string>Trial Balance</string>
   </property>
  </action>
  <action name="actionAdjustments_2">
   <property name="text">
    <string>Adjustments</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionAudit_adjustments">
   <property name="text">
    <string>Audit Adjustments</string>
   </property>
  </action>
  <action name="actionAudit_Trial_Balance">
   <property name="text">
    <string>Audit Trial Balance</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
